src/bytedeck_summernote/widgets.py

```diff
@@ -0,0 +1,93 @@
+import bleach
+import html as htmllib
+import json
+
+from django.conf import settings as django_settings
+from django.forms.utils import flatatt
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+from bleach.css_sanitizer import CSSSanitizer
+from django_summernote.utils import get_config
+from django_summernote.widgets import SummernoteInplaceWidget, SummernoteWidget
+
+
+class ByteDeckSummernoteAdvancedWidget(SummernoteWidget):
```

Comment:
Docstring needed for this class describing the purpose of the AdvancedWidget (what does it add/change from it's base class?)